### DISCUSSION TITLE
what i do

### DISCUSSION THREAD

**Discussion by rwg100** [3kh0/ext-remover#387] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
i made poll asking what i use to nuke 3kho by i forgor f15 costs 138 million dollas
what i do

**Comment by Aka-but-nice** [comment, marked as answer]
hold 3kh0 for ransom

**Reply by rwg100** [reply]
i need dyknow killed

**Reply by Aka-but-nice** [reply]
There's not a tinyurl.com/ dyknow link.
